lib/v2/udn/breaking-news.js

```diff
@@ -0,0 +1,66 @@
+const got = require('@/utils/got');
+const { parseDate } = require('@/utils/parse-date');
+const { art } = require('@/utils/render');
+const path = require('path');
+const timezone = require('@/utils/timezone');
+const cheerio = require('cheerio');
+
+module.exports = async (ctx) => {
+    const id = ctx.params.id;
+    const link = `/news/breaknews/1/${id}#breaknews`;
+    const name = await getLinkName(ctx, link);
+    const rootUrl = 'https://udn.com/';
```

Comment:
```suggestion
    const rootUrl = 'https://udn.com';
```
So response url and `link` won't becomes https://udn.com//api/more......